WordPress/src/main/res/layout/stats_block_legends_purple_item.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="@dimen/margin_extra_large"
+    android:layout_marginEnd="@dimen/margin_extra_large">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/legend1"
+        style="@style/StatsLegendsPurple"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:importantForAccessibility="no"
+        android:text="@string/stats_timeframe_this_week"
+        app:layout_constraintEnd_toStartOf="@id/legend2"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"/>
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/legend2"
+        style="@style/StatsLegendsGray"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:importantForAccessibility="no"
+        android:text="@string/stats_timeframe_previous_week"
+        app:layout_constraintStart_toEndOf="@id/legend1"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"/>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
The only difference between `stats_block_legends_blue_item` and `stats_block_legends_purple_item` is the style of `legend1`. Can we use one layout and set the style differently?

Reply:
I tried, it wasn't working.  The legend is a compound drawable on TextView.  Would look for a better way later.